src/aliases.jl

```diff
@@ -1,3 +1,26 @@
+import AbstractAlgebra: LaurentMPolyRing  # FIXME: remove these imports once AA exports these
```

Comment:
There is already an AbstractAlgebra release with them, right? So why don't drop the here and now?

Reply:
Yeah I just noticed it, too -- fixed now!